packages/skill-lesson/hooks/use-mux-player.tsx

```diff
@@ -31,6 +31,7 @@ type VideoContextType = {
   canShowVideo: boolean
   loadingUserStatus: boolean
   ability: AppAbility
+  refetchAbilityRules: () => {}
```

Comment:
how does one type this thing? seemed pretty complex so I avoided it but I might be missing something.

Reply:
one way this would "just work" is to add the subscriber to the input of the existing trpc call so then when it mutates so do abilities?